dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,339 @@
-              }"
+              type="hidden"
+              name="companyId"
+              label="Company ID"
+              placeholder="Company ID"
+              :model-value="companyID"
+              disabled="true"
             />
-          </FormKit>
-          <FormKit
-            type="radio"
-            name="reportingObligation"
-            label="Reporting Obligation"
-            :options="['Yes', 'No']"
-            :outer-class="{
-              'formkit-outer': false,
-            }"
-            :inner-class="{
-              'formkit-inner': false,
-            }"
-            :input-class="{
-              'formkit-input': false,
-              'p-radiobutton:': true,
-            }"
-          />
-          <FormKit type="group" name="eligibilityKpis" label="Eligibility KPIs">
-            <template
-              v-for="fsType in ['CreditInstitution', 'InsuranceOrReinsurance', 'AssetManagement', 'InvestmentFirm']"
-              :key="fsType"
-            >
-              <div :name="fsType">
-                <FormKit type="group" :name="fsType">
-                  <h4>Eligibility KPIs ({{ humanizeString(fsType) }})</h4>
-                  <DataPointFormElement name="taxonomyEligibleActivity" label="Taxonomy Eligible Activity" />
-                  <DataPointFormElement name="taxonomyNonEligibleActivity" label="Taxonomy Non Eligible Activity" />
-                  <DataPointFormElement name="derivatives" label="Derivatives" />
-                  <DataPointFormElement name="banksAndIssuers" label="Banks and Issuers" />
-                  <DataPointFormElement name="investmentNonNfrd" label="Investment non Nfrd" />
-                </FormKit>
+            <div class="uploadFormSection grid">
+              <div class="col-3 p-3 topicLabel">
+                <h4 id="reportingPeriod" class="anchor title">Reporting Period</h4>
+              </div>
+              <div class="col-9 formFields uploaded-files">
+                <UploadFormHeader
+                  data-test="reportingPeriodLabel"
+                  :name="euTaxonomyKpiNameMappings.reportingPeriod"
+                  :explanation="euTaxonomyKpiInfoMappings.reportingPeriod"
+                />
+                <div class="md:col-6 col-12 p-0">
+                  <Calendar
+                    data-test="reportingPeriod"
+                    v-model="reportingPeriod"
+                    inputId="icon"
+                    :showIcon="true"
+                    view="year"
+                    dateFormat="yy"
+                  />
+                </div>
+
+                <FormKit type="hidden" v-model="reportingPeriodYear" name="reportingPeriod" />
+              </div>
+              <FormKit type="group" name="data" label="data">
+                <UploadReports
+                  ref="UploadReports"
+                  :filesToUpload="filesToUpload"
+                  :uploadFiles="uploadFiles"
```

Comment:
This got Florian and me very confused naming-wise.
We should definitely think about the naming here.

`filesToUpload` is ok for me,
but `uploadFiles` does not actually contain files. So maybe smth like `listOfExtendedReportInfo" or something similar?

Reply:
Yes, right. "listOfUploadedReportsInfo" would be ok?

Reply:
Yes that's good. Thx